client/app/cloud/project/compute/infrastructure/privateNetwork/addVRack/cloud-project-compute-infrastructure-privateNetwork-addVRack.controller.js

```diff
@@ -0,0 +1,82 @@
+class AddVRackCtrl {
```

Comment:
When I open this file with ESlint on I see lot of errors which leads to think that your ESlint isn't configured correctly

Reply:
I configured ESLint properly. I can see them too. I will fix all of these.